data/reference.json

```diff
@@ -103,6 +103,13 @@
         },
         "type": "boolean"
     },
+    "function_anonymous.spacing_to_group": {
+        "description": "Whether to space a function name from following group.",
+        "example": {
+            "code": "if (true){}"
```

Comment:
Could you update the example to match the setting?

Reply:
Updated the example to represent the new functionality.